css/banners.css

```diff
@@ -106,28 +91,45 @@ html, body{
   text-decoration: none;
 }
 
+.click-area {
+  width: 100%;
+  height: 5em;
+}
+
+.arrow {
+  transition: 0.2s ease all;
+}
+
+.rotate {
+  -moz-transform: rotate(-180deg);
+  -webkit-transform: rotate(-180deg);
+  -ms-transform: rotate(-180deg);
+  -o-transform: rotate(-180deg);
+  transition: 0.2s ease all;
+}
+
 .expand {
-   height: 450px !important;
-   -webkit-transition: height 0.5s ease-in-out;
-    -moz-transition: height 0.5s ease-in-out;
-    -o-transition: height 0.5s ease-in-out;
-    transition: height 0.5s ease-in-out;
+  height: 28em !important;
+  -webkit-transition: height 0.5s ease-in-out;
+  -moz-transition: height 0.5s ease-in-out;
+  -o-transition: height 0.5s ease-in-out;
+  transition: height 0.5s ease-in-out;
 }
 
 .excerpt {
   display: inherit !important;
   visibility: visible;
   transition: visibility 0s, opacity 0.5s linear;
-
 }
 
 .excerpt p{
```

Comment:
you can nest these kinds of styles now:
```
.excerpt {
  ...
  p {
    ...
  }
}
```
try it with merging this block with the above one